src/license_manager_agent_ops.py

```diff
@@ -27,49 +28,77 @@ class LicenseManagerAgentOps:
     EPILOG_PATH = _VENV_DIR / "bin/slurmctld_epilog"
     _SLURM_USER = "slurm"
     _SLURM_GROUP = "slurm"
+    _LICENSE_MANAGER_USER = "license-manager"
+    _LICENSE_MANAGER_ACCOUNT = "license-manager"
+
 
     def __init__(self, charm):
         """Initialize license-manager-agent-ops."""
         self._charm = charm
 
     def setup_cache_dir(self):
         """Set up cache dir."""
-        CACHE_DIR = Path("/var/cache/license-manager")
-
         # Delete cache dir if it already exists
-        if CACHE_DIR.exists():
-            logger.debug(f"Clearing cache dir {CACHE_DIR.as_posix()}")
-            rmtree(CACHE_DIR, ignore_errors=True)
+        if self._CACHE_DIR.exists():
+            logger.debug(f"Clearing cache dir {self._CACHE_DIR.as_posix()}")
```

Comment:
specially when debugging, I feel like this log can be a little bit confuse. I suggest something like:
```suggestion
            logger.debug(f"The cache directory already exists. Clearing it: {self._CACHE_DIR.as_posix()}")
```